novawallet/Modules/AssetsSearch/Swaps/SwapAssetsOperationViewFactory.swift

```diff
@@ -21,6 +21,25 @@ enum SwapAssetsOperationViewFactory {
         )
     }
 
+    static func createSelectReceiveTokenView(
```

Comment:
I wouldn't name it like SelectReceive/Pay token view. I think we just can have one function that accepts optional ChainAsset model.